controller/analytics.py

```diff
@@ -1,7 +1,7 @@
 from flask import Blueprint, render_template, session
 from models.user import User
```

Comment:
why you import user?

Reply:
im using the User model in 'analytics_bp' to fetch user data from the DB 
users = await User.find().to_list() this line is used to query the db for all the users, so i think i need to import user no ??

Reply:
and i think i need to change the name analytics to something else, analytics =ניתוח and that's not what the page does

Reply:
this shouldnt look like this you should use a function like get_all_users() however this function is in your dal lets refactor the architecture a little:

├── dal/
│   ├── entities/
│   │   └── user.py --- your pydantic basemodel
│   ├── repositories/
│   │   └── user_repository.py --- crud operations function such as create_user, get_user_by_id..
│   └── database_connection.py --- from beanie import init_beanie async def connect_to_db etc...

├── models/
│   └── user.py --- same functions like user_repository but instead of working with db using user repository might seem redundant to have similar functions in both user repository and user model, each class serves a distinct purpose..

user model:
Acts as an intermediary between the business logic and data access layer.

user repository:
Handles the database interactions directly, focused on CRUD operations and database-specific logic.

Reply:
Also remove UserModel in the model why is it even there you use class user instead

Reply:
check now if u can